### DISCUSSION TITLE
A bird in mid flight!

### DISCUSSION THREAD

**Discussion by lee-costa** [blakeblackshear/frigate#2899] [Show and tell]
Just wanted to share a snapshot captured by Frigate today! I couldn't believe my eyes when I saw it! Cropped it for privacy.

![bird](https://user-images.githubusercontent.com/22748735/156820697-f91b8a87-900b-4191-9efb-3f612542f3e1.png)

**Comment by NickM-27** [comment]
That's surprisingly clear for a fast moving bird, very cool!

**Comment by ehn** [comment]
Here is another one:

![snapshot-1646202210 763151-qyl9c7](https://user-images.githubusercontent.com/179446/157034493-d8e44378-af01-47a9-865f-6358c38f3ed5.jpg)

**Comment by McPaHa** [comment]
LOL... happens all the time (from my summer cottage):
![snapshot-1646730925 163536-8iku6h](https://user-images.githubusercontent.com/36406452/157236235-4fabd4c5-e353-4f3c-a03d-0a3882188102.jpg)
![snapshot-1646730840 573844-857bm4](https://user-images.githubusercontent.com/36406452/157236251-7879f8a9-8021-482e-adf2-06ef67c47d65.jpg)
